src/sql/workbench/common/constants.ts

```diff
@@ -71,3 +71,8 @@ export const RESULTS_GRID_DEFAULTS = {
 	cellPadding: [5, 8, 4],
 	rowHeight: 24
 };
+
+export const azuremonitorLogsProviderName = 'AZUREMONITORLOGS';
```

Comment:
It would be preferable to have a single string enum of known provider IDs I think (and same for language modes)